go/pkg/settings/user.go

```diff
@@ -82,9 +82,9 @@ func (s *UserSettings) Save() error {
 }
 
 func UserSettingsDir() (string, error) {
-	return homedir.Expand("~/.config/replicate")
+	return homedir.Expand("~/.config/keepsake")
 }
 
 func userSettingsPath() (string, error) {
-	return homedir.Expand("~/.config/replicate/settings.json")
+	return homedir.Expand("~/.config/keepsake/settings.json")
```

Comment:
`~/.config/replicate` should continue to be supported, or migrated.

Reply:
Great catch, fixing.